logger/logger.go

```diff
@@ -79,7 +79,7 @@ func Default() Logger {
 func Debug(gokaDebug, saramaDebug bool) {
 	defaultLogger.debug = gokaDebug
 	if saramaDebug {
-		SetSaramaLogger(&std{debug: true})
+		SetSaramaLogger(&std{debug: true, prefix: "[Sarama] "})
```

Comment:
@adw1n  Actually I'd like some change, if you don't mind, because this would be inconsistent with how this prefixing approach works.
The `std`-logger field prefix represents the current prefix in brackets e.g. `["Sarama"]`and a prefix path, which allows to extend the prefix, so the prefix path needs to be without brackets. In this case it would need to be `[]string{"Sarama"}`. So when adding a new prefix, e.g. `log.Prefix("Consumer")` we would have a new prefix like `[Sarama>Consumer]`.

If we initialize it like in the PR, the prefix stacking would be broken, and adding a new prefix would produce a prefix like `[[Sarama]>NewPrefix]`. 
This is not very relevant at the moment, because sarama won't call `Prefix` of course, still I think it should be consistent.
To avoid duplication, the `Prefix`-function you could use the Prefix-function after initalization. Like this:
```
SetSaramaLogger((&std{debug: true}).Prefix("Sarama"))
```

What do you say?

Reply:
@frairon I agree. I commited and pushed the requested change.